test/functional/test_framework/netutil.py

```diff
@@ -25,6 +25,10 @@
 STATE_LISTEN = '0A'
 # STATE_CLOSING = '0B'
 
```

Comment:
nit
```suggestion

# Address manager size constants as defined in addrman_impl.h
```

Reply:
thanks! resolved in [380130d](https://github.com/bitcoin/bitcoin/pull/28176/commits/380130d9d70f8f8d395949a51f43806f6e600efa)